site/content/en/docs/_index.md

```diff
@@ -1,8 +1,29 @@
 ---
-title: "Documentation"
+title: "LeaderWorkerSet"
 linkTitle: "Documentation"
 weight: 20
 menu:
   main:
     weight: 20
----
\ No newline at end of file
+---
+
+LeaderWorkerSet (LWS) is an API for deploying a group of pods as a unit of replication. It aims to address common deployment patterns of AI/ML inference workloads, especially multi-host inference workloads where the LLM will be sharded and run across multiple devices on multiple nodes.
+The initial design and proposal can be found at: <http://bit.ly/k8s-LWS>.
+
+Get started with [installation](installation/) or watch the LWS-related [talks & presentations](https://lws.sigs.k8s.io/docs/adoption/#talks-and-presentations) to learn more.
+
+## Conceptual View
+
+![image](../../images/concept.png)
```

Comment:
HI @chewong 
Does `/images/concept.png` or  `../images/concept.png` is better :-)